Polygon_mesh_processing/include/CGAL/Rigid_triangle_mesh_collision_detection.h

```diff
@@ -49,26 +49,26 @@ namespace CGAL {
  * \ingroup PkgPolygonMeshProcessing
  * This class provides methods to perform some intersection tests between triangle meshes
  * that undergo affine transformations (rotation, translation, and scaling).
- * Meshes are added to an internal pool and are referenced using an id assigned when added to the pool.
+ * Meshes are added to an internal set and are referenced using an id assigned when added to the set.
  * Note that the exact predicate framework applies on the meshes after having applied the transformation
  * to the coordinates of the points of the vertices of each mesh.
  *
  * @tparam TriangleMesh a model of `HalfedgeListGraph` and `FaceListGraph`
  * @tparam VertexPointMap a model of `ReadablePropertyMap` with the vertex descriptor of `TriangleMesh` as key type,
  *                        and a point from a CGAL Kernel as value type. %Default is the internal point property map
  *                        of `TriangleMesh` if it exists.
- * @tparam KernelType a model of CGAL Kernel. %Default is the Kernel of the value type of `VertexPointMap` retrieved using
+ * @tparam Kernel a model of CGAL Kernel. %Default is the kernel of the value type of `VertexPointMap` retrieved using
  *                `Kernel_traits`.
- * @tparam AABBTreeType an `AABB_tree` that can containing faces of `TriangleMesh`. %Default is using `AABB_traits` with
+ * @tparam AABBTree an `AABB_tree` that can containing faces of `TriangleMesh`. %Default is using `AABB_traits` with
  *                       `AABB_face_graph_triangle_primitive` as primitive type.
  * @tparam HAS_ROTATION tag indicating whether the transformations applied to meshes may contain rotations (`Tag_true`)
  *                      or if only translations and scalings are applied (`Tag_false`). Some optimizations are
  *                      switch on in case there are no rotations.
  */
 template <class TriangleMesh,
           class VertexPointMap = Default,
-          class KernelType = Default,
-          class AABBTreeType = Default,
+          class Kernel = Default,
+          class AABBTree = Default,
           class HAS_ROTATION = CGAL::Tag_true>
```

Comment:
`HAS_ROTATION` does not respect our naming guidelines. I should be `Has_rotation` or `Has_rotation_tag`.